docs/cmdlet-example/command-line-simple-example.md

```diff
@@ -0,0 +1,135 @@
+# Creating a Simple Cross-Platform Binary Module with the Dotnet CLI
+
+This example shows how you can create a simple, binary module using the dotnet
+CLI (command line interface).  For this example, you can use any OS that the
```

Comment:
my preference is to Use the full name and then parenthesize the initialism `Command Line Interface (CLI)` when bring an initialism into context.

Reply:
fixed